### PR TITLE
Add model-viewer block for the Web Components docs page

### PR DESCRIPTION
**Pull request by bdelacretaz**
Add model-viewer block.

https://main--helix-website--adobe.aem.page/developer/web-components

vs

https://main--helix-website--bdelacretaz.hlx.page/developer/web-components

## Description

This adds a `model-viewer` block for the upcoming Web Components docs page, along with the example 3D viewer component from https://modelviewer.dev/ and a `<lazy-loader>` Web Component to load all of that in the `delayed` phase.

## Related Issue

#457

## How Has This Been Tested?

Besides visual testing, the new code is covered by automated tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) (I'm an Adobe employee)
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: tests/blocks/model-viewer/model-viewer.mock.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      <div>
+        <div>the-model-name</div>
+      </div>
+      <div>
+        <div>The model's description</div>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
```
Comment (dylandepass) [inline tests/blocks/model-viewer/model-viewer.mock.html:13]: nit: extra `</div>`

**Review by dylandepass — APPROVED** [review]
LGMT

**Review by kptdobe — CHANGES_REQUESTED** [review]
I think the web component sample we use should be a project specific element, not something completely unrelated for which the code does not belong to the current project code.

**Comment by bdelacretaz** [comment]
I'll close this PR and create another one with a web component that does something useful for the website.

**Pull request closed without merging**